src/sage/schemes/elliptic_curves/hom_composite.py

```diff
@@ -120,33 +118,98 @@ def _eval_factored_isogeny(phis, P):
     return P
 
 
-def _compute_factored_isogeny_prime_power(P, l, e):
-    """
-    This method takes a point `P` of order `l^e` and returns
-    a sequence of degree-`l` isogenies whose composition has
+def _compute_factored_isogeny_prime_power(P, l, n, split=.8):
+    r"""
+    This method takes a point `P` of order `\ell^n` and returns
+    a sequence of degree-`\ell` isogenies whose composition has
     the subgroup generated by `P` as its kernel.
 
+    The optional argument ``split``, a real number between
+    `0` and `1`, controls the *strategy* used to compute the
+    isogeny: In general, the algorithm performs a number of
+    scalar multiplications `[\ell]` and a number of
+    `\ell`-isogeny evaluations, and there exist tradeoffs
```

Comment:
Exactly speaking, the performed scalar multiplication is not always by $\ell$, but by some powers of $\ell$ (though the latter itself might still perform successive multiplications by $\ell$).

Reply:
Well, for the level of detail considered in this high-level description, the cost of $[\ell^n]$ is the same as the cost of $[\ell]^n$, so we may assume things are done as stated in your parenthetical remark and then the description is correct. I think the current phrasing is a little bit easier to grasp than the "exactly speaking" one, but I don't have a strong opinion about it.

Reply:
OK. I agree.